examples/poc/hadr_deployment/README.md

```diff
@@ -12,13 +12,24 @@ This deployment consists of:
 5. One Secondary Gateway
 6. Federation
 7. Hub HADR setup
-8. Gateways HADR setup
-9. Creation of a new “Demo DB” - RDS MySQL
-10. Auto configuration of the RDS MySQL to enable native audit
-11. Onboarding the DB to DSF Hub
-12. There is also an option to create an RDS MsSQL with audit configured and with synthetic data on it. In order to do so, use the variable 'db_types_to_onboard' and specify which DBs to create and onboard.<br/>
+8. Creation of a new “Demo DB” - RDS MySQL 
+9. Auto configuration of the RDS MySQL to enable native audit 
+10. Onboarding the DB to DSF Hub 
+11. There is also an option to create an RDS MsSQL with audit configured and with synthetic data on it. In order to do so, use the variable 'db_types_to_onboard' and specify which DBs to create and onboard.<br/>
     For example, in order to create both 'RDS MySQL' and 'RDS MsSQL', run the following:
    ```bash
    terraform apply -auto-approve -var 'db_types_to_onboard=["RDS MySQL", "RDS MsSQL"]'
    ```
-   For more details, go to the rds-mssql-db module.
\ No newline at end of file
+   For more details, go to the rds-mssql-db module.
+
+## Default Example
+```bash
+terraform apply -auto-approve
+```
+
+## Customized Example
+The default example contains variables with default values. In order to customize the variables, you can use the following for example:
```

Comment:
The word 'example' is used with two different meanings in this sentence. I suggest: "you can use the following for instance:"